packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2430,8 +2460,32 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
   }
 
   private outputTrackUpdateListener(localAudioStream: LocalMicrophoneStream) {
-    localAudioStream.on(LocalStreamEventNames.OutputTrackChange, (track: MediaStreamTrack) => {
-      this.mediaConnection.updateLocalTracks({audio: track});
+    let effect: any;
```

Comment:
Does web-media-effects not export any generic Effect type?

Reply:
Can we not use TrackEffect for this?

Reply:
If I use TrackEffect, there are a lot of types errors